src/components/MainView/ListOfDashboards/ListOfDashboards.scss

```diff
@@ -0,0 +1,10 @@
+.lod-container {
+  display: flex;
+  flex: 0 0 auto;
+  justify-content: space-around;
+  flex-wrap: wrap;
+  padding-bottom: 10px;
+  min-height: 0 !important;
```

Comment:
is 'important' really needed here?

Reply:
I am not engaged in styles improvement - this is Oksana's task, so my goal is to make it work somehow. no 'important!' makes this task impossible because of the style dependencies Oksana uses